templates/typesetting/typesetting_upload_image_zip.html

```diff
@@ -0,0 +1,25 @@
+{% extends "admin/core/base.html" %}
+
+{% load files %}
+
+{% block body %}
+    <div class="box">
+        <div class="title-area">
+            <h2>Upload a Zip File</h2>
+        </div>
+        <form method="POST" enctype="multipart/form-data">
+            {% csrf_token %}
+            <p>
+                Your zip file should have a series of images with the same names
+                as the images in your XML file. If a duplicate is found it will be updated with the new image.
+            </p>
+            <ul>
+                {% for image in galley.all_images %}
+                <li>{{ image }}</li>
+                {% endfor %}
+            </ul>
+            <input name="file" type="file">
+            <button class="small success button" name="zip_file">Upload</button>
+        </form>
+    </div>
+{% endblock body %}
```

Comment:
Why was this template committed with the recent heading change?

Reply:
Removed.